db/migrate/20150116172420_teams_members_cleanup.rb

```diff
@@ -0,0 +1,9 @@
+class TeamsMembersCleanup < ActiveRecord::Migration
```

Comment:
this migration can be removed. 
Could you add a validation if not present in the Teams::Member model instead ?